lib/rubocop/cop/lint/uri_escape_unescape.rb

```diff
@@ -53,23 +55,31 @@ class UriEscapeUnescape < Cop
 
         def on_send(node)
           uri_escape_unescape?(node) do |top_level, obsolete_method|
-            replacements = if %i[escape encode].include?(obsolete_method)
-                             ALTERNATE_METHODS_OF_URI_ESCAPE
-                           else
-                             ALTERNATE_METHODS_OF_URI_UNESCAPE
-                           end
-
             double_colon = top_level ? '::' : ''
 
             message = format(
               MSG, uri_method: "#{double_colon}URI.#{obsolete_method}",
-                   replacements: "`#{replacements[0]}`, `#{replacements[1]}` " \
-                                 "or `#{replacements[2]}`"
+                   replacements: replacements_string(obsolete_method)
```

Comment:
happy to change this if desired. we could do something similar to some of the suggestions here https://codereview.stackexchange.com/questions/5863/ruby-function-to-join-array-with-commas-and-a-conjunction/5890